index.js

```diff
@@ -56,6 +56,10 @@ module.exports = function (PIXI)
         resource.data.spritesheets.forEach(function(spritesheet) {
             var name = spritesheet.image;
             var imageUrl = route + "/" + spritesheet.image;
+            PIXI.Texture.removeFromCache(name);
+            PIXI.Texture.removeFromCache(imageUrl);
+            PIXI.BaseTexture.removeFromCache(name);
+            PIXI.BaseTexture.removeFromCache(imageUrl);
```

Comment:
why do you need to remove it from both the base texture and texture? aren't they the same?

Reply:
surprisingly, not

Reply:
"lol"

Reply:
for realz?